### ISSUE TITLE
Streamlined Login, Interactive Blogs, and Enhanced UI Design

### ISSUE THREAD

**Issue by TanmayAggarwal87** [OkenHaha/react-blog#19]
I propose implementing a custom signup and login system using MongoDB, providing users with enhanced flexibility and a better user experience. This feature will allow seamless integration with the database, ensuring that all user accounts and blogs are managed through a centralized system. Additionally, we can introduce a commenting feature for each blog, fostering engagement and interaction.

To enhance the visual appeal of the platform, Multer can be utilized for uploading cover images for blogs. This will significantly improve the presentation of each post and make the platform more engaging for users.

For the main UI, we suggest incorporating small blog cards that display the cover image along with the creator's details. This design will improve the user experience by making the content more accessible and visually organized

Ps: I have also created my own blogging app so I am more familiar with the code and setup

**Comment by OkenHaha** [comment]
@TanmayAggarwal87 Make sure that your proposed issue isn't AI generated because it does look like to me as your description (not judging! Using AI is a great way to speed up process and discussion, everyone uses it nowdays) describes what we're trying to do with the blog app. Since contributions are to be made by everyone from beginner to intermediate, let's keep approaching issues or ideas small steps at a time. In the mean time, feel free to look for existing issues you like to work, wait for new ones to pop up or find some minor or mid issue that you can come up with and like to work!

**Comment by TanmayAggarwal87** [comment]
Yeah its write that this proposal is AI generated and that's because it was my first time raising an issue to aa Github repo so I wasn't familiar with the way of proposing my ideas  that's why I let AI handle it.

Anyway thanks for the response will keep that in mind.

**Comment by OkenHaha** [comment]
> because it was my first time raising an issue to a Github repo so I wasn't familiar with the way of proposing my ideas  that's why I let AI handle it.

It's all good! It's a good idea that you made the decision to use AI for help. It's even more awesome to join the SWOC discord community and ask for help from fellow contributors too! Try looking around the code or the website and see if you can find ideas!
